### PR TITLE
update dependencies.

### PR DESCRIPTION
**Pull request by rogpeppe**
Fix a test failure caused by the blobstore update. It no longer
fails when given a mismatched file size.

### CODE REVIEW

File: internal/charmstore/addentity_test.go [hunk 1]
```diff
@@ -195,7 +195,7 @@ var uploadEntityErrorsTests = []struct {
 	url:         "~charmers/precise/wordpress-0",
 	upload:      storetesting.NewCharm(nil),
 	blobSize:    99999,
-	expectError: "cannot put archive blob: cannot calculate data checksums: EOF",
+	expectError: "cannot read charm archive: seek past end of file",
```
Comment (mhilton) [inline internal/charmstore/addentity_test.go:198]: Probably not for this PR, but can we now support Chunked requests when uploading?

**Comment by mhilton** [comment]
:+1:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1645/
Test PASSed.

**Comment by fabricematrat** [comment]
:+1:

**Comment by rogpeppe** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Pull request merged**